src/cm.css

```diff
@@ -498,35 +490,37 @@ a:focus-visible {
 /** Container that is wrapped around the modal */
 .modal-container {
   position: fixed;
-  bottom: var(--modal-margin);
-  max-height: calc(100vh - var(--modal-margin) * 2);
-  left: var(--modal-margin);
-  right: var(--modal-margin);
+  bottom: 10px;
+  left: 5px;
+  right: 5px;
   margin: 0 auto;
-  width: calc(100% - var(--modal-margin) * 2);
+  width: calc(100% - 10px);
   box-sizing: border-box;
   border-radius: 18px;
   border: 0.5px solid #d8d8d8;
   background-color: rgba(255, 255, 255, 0.9);
   overflow: hidden;
-  padding: var(--modal-padding);
-  display: flex;
-  flex-direction: column;
+  padding: 21px;
   font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir,
     helvetica neue, helvetica, 'sans-serif';
   box-shadow: 0 5px 10px rgba(0, 0, 0, 0.1);
   backdrop-filter: blur(20px) saturate(5);
   animation: fadeIn 150ms;
+  max-height: 90%;
   transition: width 150ms, height 150ms;
 }
 
 /** Inner contents wrapper for modal content */
 .modal-container-inner {
   min-height: 79px;
-  display: flex;
+  height: calc(100% - 38px);
   width: 100%;
-  height: 100%;
-  justify-content: center;
+  padding-bottom: 38px;
+}
+
+/** Styles that wrap the text container of the modal content. Excludes footer */
+.inner-container {
+  overflow-y: auto;
```

Comment:
This is the only line different from v2.10.3. This PR is technically just a single line change + a bunch of reverts

Reply:
@eligrey did you confirm that this fixes the issue?

Reply:
build is failing with this error, is it expected?

![Screen Shot 2023-01-11 at 11 32 34 AM](https://user-images.githubusercontent.com/12694524/211862398-5422b091-b7e5-4e2b-a90f-198511b78e2b.png)

Reply:
Weird. I didn't get that error last night. I'll probably just un-revert the tslib  .js change to fix this